code/modules/urist/antagonist/revenant/distortion_effects.dm

```diff
@@ -0,0 +1,125 @@
+// Various spoopiness that happens when too much ambient Distortion accumulates.
+
+
+/datum/power/revenant/distortion
+	isVerb = FALSE // for convenience, as most won't be
+	verbpath = null // we'll use Apply() for most cases; keeping this for stuff affecting BsRev (adding callbacks etc.)
+
+
+var/list/revenant_distortions = (typesof(/datum/power/revenant/distortion) - /datum/power/revenant/distortion)
```

Comment:
this should just be a var of /datum/bluespace_revenant, should it not?

Reply:
Well, it _could_ be, but I'm not altogether sure it _should_. 

This is a reflection singleton that might be plugged into procs etc. outside of the class itself.

The problem is datums are a PITA to query convenience-wise, and on top of that it's pointer-chasing which is a relatively slow operation (unless the DM VM bullshit handles it nicely somehow, but given that it's DM we're talking about... I doubt it).

So it's both slower and less convenient to have it as an attribute, and ultimately it doesn't really _hurt_ anyone to have it not be one since it's pure data.

Reply:
fair enough, but in that case it'll need to be made into a global to appease the tests.